script/create_test_db.sh

```diff
@@ -1,89 +0,0 @@
-#!/bin/bash
```

Comment:
As mentioned in https://github.com/metabrainz/musicbrainz-server/pull/523#discussion_r120760834, this file should be left alone, and you should decide which script to call in docker/musicbrainz-test-database/create_test_db.sh based on the value of `$MB_IMPORT_DUMPS` (which is set on the container via `docker run --env`).